### PR TITLE
sage: 8.6 -> 8.7

### PR DESCRIPTION
**Pull request by timokau**
###### Motivation for this change

Here's the next round. Tracking the betas and rc's until the next release (if I get around to it).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by timokau** [comment]
Just as a status update: sage 8.7 has been released, but the update in nix is blocked by https://github.com/NixOS/nixpkgs/pull/56911 which is in turn blocked by https://github.com/aleaxit/gmpy/issues/199.

File: pkgs/development/python-modules/pplpy/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/pplpy/default.nix]:
```suggestion
    gmpy2
```

File: pkgs/top-level/python-packages.nix
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix]: This is no longer needed.

**Review by dotlambda — DISMISSED** [review]

File: pkgs/top-level/python-packages.nix
Comment (timokau) [reply pkgs/top-level/python-packages.nix]: Right, thanks for noticing.

**Comment by timokau** [comment]
The gmpy2 beta release seems to be a while off yet. I don't want to hold back the sage release any longer and gmpy2 only has a handful of reverse dependencies. I'll merge if a final `nix-review` passes for this PR.

**Comment by timokau** [comment]
Next iteration in https://github.com/NixOS/nixpkgs/pull/59989.

**Pull request merged**
